netsuite/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+
+NetSuite's [SuiteTalk SOAP API](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml) provides capability to access NetSuite operations related different kind of NetSuite records
+such as Account, Client, Transactions, Invoice, Classifications etc.
+
+This module supports [NetSuite WSDL 2020.2.0](https://system.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2020_2/schema/record/account.html) version.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* A NetSuite account
+* Obtaining tokens
+1. Go to [NetSuite](https://www.netsuite.com) and login to your account.
+2. If you have NetSuite Permission to Create Integration APP, The following steps may be helpful or else go to step 03.
+    1. Enable the SuiteTalk Web service features of the account (Setup->Company->Enable Features).
+    
+    2. Create an integration application (Setup->Integration->New), enable TBA code grant and scope, and obtain the 
+    following credentials: 
+        * Client ID
+        * Client Secret
+3. If you have Client ID, Client Secret from your administrator, Obtain the below credentials by following the token based authorization in the [NetSuite documentation](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml). 
```

Comment:
Suggestion: "If you have a client ID and a client secret from your administrator, obtain. the credentials below by following the instructions for token-based authorization in the [NetSuite documentation](https://www.netsuite.com/portal/developers/resources/suitetalk-documentation.shtml)."